cypress/support/step_definitions/dictionary/addBulkConceptsToDictionary/addBulkConceptsToDictionary.ts

```diff
@@ -0,0 +1,128 @@
+/// <reference types="cypress" />
+/// <reference types="../../../" />
+import { 
+  After, 
+  Before, 
+  Given, 
+  Then, 
+  When } from "cypress-cucumber-preprocessor/steps";
+import { getDictionaryId, getUser, getConceptId } from '../../../utils';
+
+Given("the user is on the view dictionary concepts page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/`);
+  cy.url().should(
+    "contain",
+    `/users/${getUser()}/collections/${getDictionaryId()}/concepts/`
+  );
+});
+
+Given('the user is on the "Add concepts in bulk from CIEL" page', () => {
+cy.visit(
+`/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+);
+cy.url().should(
+"contain",
+`/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+  );
+});
+
+Given('CIEL concept "1000" is already in the dictionary', () => {
+  cy.visit('/actions/');
+  cy.url().should(
+    "contain", '/actions/'
+  );
+});
+
+When('the user clicks the "Add concepts" button', () => {
+  cy.findByText("Add concepts").click({force: true});
+});
+
+When('the user selects "Add bulk concepts"', () => {
+cy.visit(
+  `/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+  );
```

Comment:
```suggestion
  cy.visit(
    `/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
  );
```

Reply:
This isn't actually doing what's described in the text, though. In the text, we're supposed to be finding the element labelled "Add bulk concepts" and click on it to be redirected to the location... this just redirects directly there.

Reply:
@ibacher and @jayasanka-sack thanks for your review and proposed changes. Let me fix them right away!

Reply:
> This isn't actually doing what's described in the text, though. In the text, we're supposed to be finding the element labelled "Add bulk concepts" and click on it to be redirected to the location... this just redirects directly there.

@ibacher am sorry for a bother! did I miss out on the location?